src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,135 @@
+			"yesBtn": "Yes",
+			"cancelBtn": "Cancel"
+
+		},
+		fr: {
+			"msgboxHeader": "Avertissement",
+			"exitMsg": "Vous êtes sur le point de quitter un site sécurisé. Voulez-vous continuer?",
+			"yesBtn": "Oui",
+			"cancelBtn": "Annuler"
+
+		}
+	},
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			settings,
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+				true,
+				window[ componentName ],
+				wb.getData( $elm, componentName )
+
+			);
+
+			$elm.data( componentName, settings );
+
+			i18nDict = i18nDict[ wb.lang || "en" ];
+
+			if ( $elm.hasClass( exiturlparam ) ) {
+				this.outerHTML = "<a href='" + sessionStorage.getItem( "originalURL" ) + "'>" + sessionStorage.getItem( "originalURL" ) + "</a>";
+				if ( document.querySelector( selector + "-targetparam" ) ) {
+					document.querySelector( selector + "-targetparam" ).outerHTML = sessionStorage.getItem( "trgAttribute" );
+				}
+			}
+			wb.ready( $elm, componentName );
+		}
+	};
+
+$document.on( "click", selector, function( event ) {
+	event.preventDefault();
+
+	var elm = event.currentTarget,
+		$elm = $( elm ),
+		moDalId,
+		wrapper,
+		moDal = document.createDocumentFragment(),
+		tpl = document.createElement( "div" ),
+		settings =  $elm.data( componentName ),
+		modalWindow = settings.displayModal !== false,
+		msgboxHeader = settings.msgboxHeader || i18nDict.msgboxHeader,
+		exitMsg = settings.exitMsg || i18nDict.exitMsg,
+		yesBtn = settings.yesBtn || i18nDict.yesBtn,
+		cancelBtn = settings.cancelBtn || i18nDict.cancelBtn,
+		exitURL = settings.exitURL,
+		targetAttribute = this.getAttribute( "target" ) ? "target='" + this.getAttribute( "target" ) + "'" : "",
+		re = new RegExp( "^(http|https)://", "i" );
+
+	// Check browser support and exit url does not begin with http:// or https://
+	if ( !re.test( exitURL ) ) {
+		sessionStorage.setItem( "trgAttribute", this.getAttribute( "target" ) );
+		sessionStorage.setItem( "originalURL", this );
+
+	}
+
+	if ( modalWindow ) {
+		tpl.innerHTML = "<section class='mfp-hide modal-dialog modal-content overlay-def'>" +
+			"<header class='modal-header'><h2 class='modal-title'>" + msgboxHeader + "</h2></header>" +
+			"<div class='modal-body'>" +
+			"<p>" + exitMsg + "</p>" +
+			"<ul class='list-inline text-center'>" +
+			"<li><button class='btn btn-primary popup-modal-dismiss pull-left' id='eCancel'>" + cancelBtn + "</button></li>" +
+			"<li><a class='btn btn-default pull-right popup-modal-close'" + targetAttribute + " href='" + exitURL + "' type='button'>" + yesBtn + "</a></li>" +
+			"</ul></div></section>";
+		moDal.appendChild( tpl );
+		wrapper = moDal.firstChild;
+		wrapper.firstChild.id = wb.getId();
+		moDalId = wrapper.firstChild.id;
+		document.body.appendChild( wrapper.firstChild );
+
+		$document.trigger( "open.wb-lbx", [
```

Comment:
You can do instead of using the global event trigger
```
$( wrapper.firstChild ).trigger( ...... )
```

Reply:
@duboisp Please review...